conf/messages

```diff
@@ -144,7 +144,8 @@ sector.hint.1=This sector includes couriers and hauliers who get paid by someone
 sector.hint.2=An ‘agricultural product’ can be products such as: live animals, meat and edible meat offal, dairy produce, birds’ eggs, natural honey. The cap for this sector is €20,000
 sector.hint.3=If any part of your business is involved in the production, processing or marketing of fishery and aquaculture products. A ‘fishery product’ means aquatic organisms resulting from any fishing activity or products derived from them. The cap for this sector is €30,000
 
-addBusinessIntent.title = Are there other businesses in your undertaking?
+addBusinessIntent.title = Businesses in your undertaking
+addBusinessIntent.legend = Are there other businesses in your undertaking?
```

Comment:
I did think about the similar in mostly duplicate in addBusinessIntent.hint but I think trying to reuse adds a different type of complication :)